commands/ci/retry/retry.go

```diff
@@ -0,0 +1,51 @@
+package run
+
+import (
+	"fmt"
+
+	"github.com/profclems/glab/api"
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/profclems/glab/pkg/utils"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdRetry(f *cmdutils.Factory) *cobra.Command {
+	var pipelineRetryCmd = &cobra.Command{
+		Use:     "retry <id>",
+		Short:   `Retry a CI job`,
+		Aliases: []string{},
+		Example: heredoc.Doc(`
+	$ glab ci retry 871528
+`),
+		Long: ``,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+
+			apiClient, err := f.HttpClient()
+			if err != nil {
+				return err
+			}
+
+			repo, err := f.BaseRepo()
+			if err != nil {
+				return err
+			}
+
+			jobID := utils.StringToInt(args[0])
```

Comment:
We should check beforehand, if the provided ID is valid. `utils.StringToInt()` returns 0 if the value isn't a valid int

```suggestion
			jobID := utils.StringToInt(args[0])
			if jobID < 1 {
				fmt.Fprintf(f.IO.StdErr, "invalid job id: %d", jobID)
				return cmdutils.SilentError
			}
```

Reply:
Thanks for the suggestion. I will display original args[0] instead of jobID in the output
```go
	if jobID < 1 {
		fmt.Fprintln(f.IO.StdErr, "invalid job id:", args[0])
		return cmdutils.SilentError
	}
```

Reply:
> I will display original args[0] instead of jobID in the output

That's even more perfect! Thanks